articles/terraform/terraform-vm-msi.md

```diff
@@ -0,0 +1,92 @@
+# Terraform with a Linux Viratul Machine on Azure
+
+This article shows you how to use a [Terraform Market Place Image](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/azure-oss.terraform?tab=Overview) to create a `Ubuntu Linux VM (16.04 LTS)` with latest `Terraform` installed and configured using [Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview). This image also configures a remote backend to enable [Remote State](https://www.terraform.io/docs/state/remote.html) management using Terraform. This VM image makes it easy to get started using Terraform on Azure in minutes, without having to install and configure Terraform and configure authentication manually. 
```

Comment:
"Terraform Market Place Image" -> "Terraform Marketplace image"

Reply:
"with latest Terraform installed" -> "with the latest Terraform version installed"

Reply:
"This image" -> "The Terraform Marketplace image"

Reply:
Done